compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -90,12 +89,31 @@ import qualified Text.PrettyPrint.ANSI.Leijen      as PP
 -- Commands
 --------------------------------------------------------------------------------
 
-cmdIde :: Mod CommandFields Command
+data CommandName = Build
+                 | Clean
+                 | Compile
+                 | DamlDoc
+                 | DocTest
+                 | Ide
+                 | Init
+                 | Inspect
+                 | InspectDar
+                 | License
+                 | Lint
+                 | MergeDars
+                 | Migrate
+                 | Package
+                 | Test
+                 | Visual
+                 deriving (Ord, Show, Eq)
+newtype Command = Command (CommandName, IO ())
```

Comment:
Newtypes around tuples are a bit weird. Usually you use `newtype` around a single type and if you have multiple fields you use `data`.